.ci/cico_updates_minikube.sh

```diff
@@ -86,7 +86,21 @@ waitCheUpdateInstall() {
   fi
 }
 
+self_signed_minikube() {
```

Comment:
We don't have to generate certificate for minikube.
It covers by che-operator itself.

Reply:
yes, you are right. I'll remove that